stable/redis/templates/init-cluster.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if and .Values.cluster.enabled ( not .Values.cluster.externalAccess.enabled )}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "redis.fullname" . }}-cluster-trigger
```

Comment:
Is the name trigger the one they use in their official documentation? cluster-create sounds better to me

Reply:
No, I invented it. 🙃